### DISCUSSION TITLE
combine handlers for 'std/http/server.ts'

### DISCUSSION THREAD

**Discussion by dagda1** [denoland/deno#15690] [Q&A]
At the moment I am creating 2 servers with [std/http/server.ts](https​://deno.land/std​@0.153.0/http/server.ts)

```ts
import { serve } from 'http/server.ts';
import { createServer } from "ultra/server.ts";
import App from "./App.tsx";

const server = await createServer({
  importMapPath: import.meta.resolve("../import_map.json"),
  browserEntrypoint: import.meta.resolve("./client.tsx"),
});

server.get('*', async (context: any) => {
// rest
}, {
  port: 8000
});

serve(server.fetch);
```

and 1 using this [handleRequest](https://github.com/dotansimha/graphql-yoga/blob/e598de7e51620d649bf961d4a34313af306bd755/packages/common/src/server.ts#L550) function from [@graphql-yoga](https://github.com/dotansimha/graphql-yoga).

```ts
import { serve } from 'http/server.ts'
import { createServer } from '@graphql-yoga/common'

export function main() {
  const graphQLServer = createServer();

  serve(graphQLServer.handleRequest, {
    port: 4000,
  });

  console.log(`Server is running on http://localhost:${PORT}/graphql`)
}
```

I would like only 1 http server.  Is it possible to combine the 2 somehow?

**Comment by bartlomieju** [comment]
These servers are listening on two completely different port so there's no way to combine them
